src/coredump/coredump.c

```diff
@@ -1212,8 +1223,23 @@ static int process_kernel(int argc, char* argv[]) {
 
         if (context.is_journald || context.is_pid1)
                 r = submit_coredump(&context, iovw, STDIN_FILENO);
-        else
+        else {
+                process_name  = filename_escape(context.meta[META_COMM]);
+                if (!process_name) {
+                        log_error_errno(r, "Failed to escape process name. Ignoring coredump ratelimit %m");
+                } else {
```

Comment:
no {} around single-line if blocks. See https://systemd.io/CODING_STYLE/#formatting

Reply:
Addressed the comment. Please check once.